.github/workflows/ubuntu-syscollector-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Syscollector test on Ubuntu
+
+on:
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Maybe we should stay with a fixed version instead of latest (like macOS) to prevent a breaking change to make the tests fail.

Reply:
The idea is to use the latest version available, in the case of macOS it is used on purpose because our agent cannot be compiled for macOS Ventura.